candi/cloud-providers/azure/terraform-modules/master-node/variables.tf

```diff
@@ -40,19 +40,20 @@ variable "clusterUUID" {
-  ssh_public_key     = var.providerClusterConfiguration.sshPublicKey
-  disk_type          = lookup(var.providerClusterConfiguration.masterNodeGroup.instanceClass, "diskType", "StandardSSD_LRS")
-  disk_size_gb       = lookup(var.providerClusterConfiguration.masterNodeGroup.instanceClass, "diskSizeGb", 50)
-  enable_external_ip = lookup(var.providerClusterConfiguration.masterNodeGroup.instanceClass, "enableExternalIP", false)
-  urn                = split(":", var.providerClusterConfiguration.masterNodeGroup.instanceClass.urn)
-  image_publisher    = local.urn[0]
-  image_offer        = local.urn[1]
-  image_sku          = local.urn[2]
-  image_version      = local.urn[3]
-  actual_zones       = lookup(var.providerClusterConfiguration, "zones", null) != null ? tolist(setintersection(["1", "2", "3"], var.providerClusterConfiguration.zones)) : ["1", "2", "3"]
-  zones              = lookup(var.providerClusterConfiguration.masterNodeGroup, "zones", null) != null ? tolist(setintersection(local.actual_zones, var.providerClusterConfiguration.masterNodeGroup["zones"])) : local.actual_zones
-  additional_tags    = merge(lookup(var.providerClusterConfiguration, "tags", {}), lookup(var.providerClusterConfiguration.masterNodeGroup.instanceClass, "additionalTags", {}))
+  prefix                 = var.clusterConfiguration.cloud.prefix
+  admin_username         = "azureuser"
+  machine_size           = var.providerClusterConfiguration.masterNodeGroup.instanceClass.machineSize
+  ssh_public_key         = var.providerClusterConfiguration.sshPublicKey
+  disk_type              = lookup(var.providerClusterConfiguration.masterNodeGroup.instanceClass, "diskType", "StandardSSD_LRS")
+  disk_size_gb           = lookup(var.providerClusterConfiguration.masterNodeGroup.instanceClass, "diskSizeGb", 50)
+  enable_external_ip     = lookup(var.providerClusterConfiguration.masterNodeGroup.instanceClass, "enableExternalIP", false)
+  urn                    = split(":", var.providerClusterConfiguration.masterNodeGroup.instanceClass.urn)
+  image_publisher        = local.urn[0]
+  image_offer            = local.urn[1]
+  image_sku              = local.urn[2]
+  image_version          = local.urn[3]
+  actual_zones           = lookup(var.providerClusterConfiguration, "zones", null) != null ? tolist(setintersection(["1", "2", "3"], var.providerClusterConfiguration.zones)) : ["1", "2", "3"]
+  zones                  = lookup(var.providerClusterConfiguration.masterNodeGroup, "zones", null) != null ? tolist(setintersection(local.actual_zones, var.providerClusterConfiguration.masterNodeGroup["zones"])) : local.actual_zones
+  additional_tags        = merge(lookup(var.providerClusterConfiguration, "tags", {}), lookup(var.providerClusterConfiguration.masterNodeGroup.instanceClass, "additionalTags", {}))
+  accelerated_networking = lookup(var.providerClusterConfiguration.masterNodeGroup.instanceClass, "acceleratedNetworking", true)
```

Comment:
This is not good. We don't want to change defaults, so all masters nodes have to be converged. I suggest using the same scheme as for yandex platform migration. First, we set default to `false` for acceleratedNetworking and then on the next release we change it to `true`. The same is about static nodes.

Reply:
The default is specified in the spec, left here just in case to avoid errors. I set a value similar to that specified in the spec, so as not to confuse.